e2e/node/e2e.test.ts

```diff
@@ -366,19 +365,28 @@ const serversUnderTest: AuthDetails[] = [
 // eslint-disable-next-line jest/no-disabled-tests
 describe.each(serversUnderTest)(
   "Authenticated end-to-end tests against Pod [%s] and OIDC Issuer [%s]:",
-  (rootContainer, oidcIssuer, clientId, clientSecret) => {
+  (rawRootContainer, rawOidcIssuer, clientId, clientSecret) => {
+    if (!rawRootContainer || !rawOidcIssuer) {
+      console.log("Skipping test due to missing env variables");
+      return;
+    }
+
     // Re-add `https://` at the start of these URLs, which we trimmed above
     // so that GitHub Actions doesn't replace them with *** in the logs.
-    rootContainer = "https://" + rootContainer;
-    oidcIssuer = "https://" + oidcIssuer;
+    const rootContainerUrl = new URL("https://" + rawRootContainer);
+    const oidcIssuerUrl = new URL("https://" + rawOidcIssuer);
+
+    const rootContainer = rootContainerUrl.toString();
+    const oidcIssuer = oidcIssuerUrl.toString();
+
     function supportsWac() {
```

Comment:
Maybe it makes sense to pass these via env vars? `E2E_TEST_FEATURES` or something, and it's a list of `wac,acp`

Reply:
Yeah, something that isn't based on domain would be idea.